lib/componentRelay.ts

```diff
@@ -290,6 +290,9 @@ export default class ComponentRelay {
 
     this.activateThemes(data.activeThemeUrls || [])
 
+    // After activateThemes is done, we want to send a message with the ThemesActivated action.
+    this.postMessage(ComponentAction.ThemesActivated, {})
```

Comment:
I'm sending the `ThemesActivated` action here and not within `this.activateThemes` because there may be editors that don't accept themes (when the componentRelay has been initialized with the `acceptsThemes` option to `false`).

Perhaps we should rename `ThemesActivated` to something like `ComponentReady`? Either way, this action will be used to tell when the component can be displayed (and the Loading screen can be hidden).

What do you think?

Reply:
Seems fine as is, but up to you.